src/components/Installer/index.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from "react";
+import styles from "./styles.module.css";
+
+export default function CondaInstallerSelector() {
+  const [os, setOs] = useState("");
+  const [arch, setArch] = useState("");
+  const [installerType, setInstallerType] = useState("");
+
+  const osArchitectures = {
+    Linux: ["x64", "aarch64", "ppc64le"],
+    macOS: ["x64", "ARM64"],
+    Windows: ["x64", "ARM64 (Beta)"],
+  };
+
+  const installerData = {
+    Anaconda: {
+      name: "Anaconda",
+      url: "https://docs.anaconda.com/free/anaconda/install/index.html",
+      defaultChannel: "defaults",
+      description: "Complete distribution for scientific Python development",
+    },
+    Miniconda: {
+      name: "Miniconda",
+      url: "https://docs.conda.io/en/latest/miniconda.html",
+      defaultChannel: "defaults",
+      description: "Minimal conda installer with Python",
+    },
+    Miniforge: {
+      name: "Miniforge",
+      url: "https://github.com/conda-forge/miniforge",
+      defaultChannel: "conda-forge",
+      description: "Community-driven minimal installer using conda-forge",
+    },
+    Mambaforge: {
+      name: "Mambaforge",
+      url: "https://github.com/conda-forge/miniforge#mambaforge",
+      defaultChannel: "conda-forge",
+      description: "Miniforge variant with mamba as default package manager",
+    },
```

Comment:
```suggestion
```

Mambaforge has been deprecated since July 2024 since it's identical to Miniforge and will stop working consistently in 2025. More details here: https://github.com/conda-forge/miniforge/blob/e733f7bbc41f42551e9f02766d8a0301b72fde26/README.md#miniforge3